v2.0/enable-node-map.md

```diff
@@ -195,11 +195,12 @@ The **Node Map** is displayed only for the locality levels that have latitude/lo
 
 ## Location Coordinates for Reference
```

Comment:
To be explicit, perhaps we should update this to `Location Coordinates for AWS and Azure`.

Reply:
Probably, although it sounds like @Amruta-Ranade would be changing it back shortly, so I'll leave it as is to avoid changing the URL location back and forth.

Reply:
SGTM.